garage/config.py

```diff
@@ -56,6 +56,8 @@
 
 FAST_CODE_SYNC_IGNORES = [".git", "data", ".pods"]
 
+GIT_REPO_URL = "https://github.com/rlworkgroup/garage.git"
```

Comment:
this belongs in config_personal_template.py

Reply:
Moved to config_personal_template.py